spec/v3.spec.json

```diff
@@ -96,13 +98,18 @@
 		  }
 		},
 		"content": {
-		  "title": "Source code",
+		  "title": "Inlined source code",
 		  "type": "string"
 		},
 		"installPath": {
 		  "title": "Target file path to install source file",
 		  "type": "string",
 		  "pattern": "^\\.\\/.*$"
+		},
+		"type": {
+		  "title": "Type",
+		  "description": "File type of this source",
```

Comment:
Is this a new field in the spec?  What types of values would go here?  What is this used for?

Reply:
[Yup](http://ethpm.github.io/ethpm-spec/v3-package-spec.html#id2).

> The type field declares the type of the source file. The field should be one of the following values: `jsonabi`, `solidity`, `vyper`.

This was a request from the compiler teams since it makes it simple for a compiler to know which files they should care about / ignore.